src/ui/resource/Image.js

```diff
@@ -28,13 +28,19 @@ import device;
 import lib.PubSub;
 import event.Callback as Callback;
 import ui.resource.loader as resourceLoader;
+import cache.LRUCache as LRUCache;
 
+var TINT_CACHE_SIZE = 4096;
```

Comment:
This seems rather large.  I'd be worried about crashes on mobile browsers when we run out of memory -- not sure how they handle textures.  We should try to test this.